src/componenets/calculator.js

```diff
@@ -0,0 +1,48 @@
+import { Component } from 'react';
+import './calculator.scss';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends Component {
+  render() {
+    return (
+      <div className="calculator-wrapper">
+        <div className="screen">
+          <p>0</p>
+        </div>
+        <div className="btn-container">
+          <div className="btn-row">
+            <button className="btn" type="button">AC</button>
+            <button className="btn" type="button">+/-</button>
+            <button className="btn" type="button">%</button>
+            <button className="btn color" type="button">&divide;</button>
+          </div>
+          <div className="btn-row">
+            <button className="btn" type="button">7</button>
+            <button className="btn" type="button">8</button>
+            <button className="btn" type="button">9</button>
+            <button className="btn color" type="button">&times;</button>
+          </div>
+          <div className="btn-row">
+            <button className="btn" type="button">4</button>
+            <button className="btn" type="button">5</button>
+            <button className="btn" type="button">6</button>
+            <button className="btn color" type="button">&minus;</button>
+          </div>
+          <div className="btn-row">
+            <button className="btn" type="button">1</button>
+            <button className="btn" type="button">2</button>
+            <button className="btn" type="button">3</button>
+            <button className="btn color" type="button">&#43;</button>
+          </div>
+          <div className="btn-last-row">
+            <button className="btn zero" type="button">0</button>
+            <button className="btn" type="button">2</button>
+            <button className="btn color" type="button">&#61;</button>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Please consider removing the white space on the right and left sides of the buttons on the calculator. It's important to try as much as possible to replicate the design provided for this Milestone, in real-world projects you'll be required to do exactly this. Please see the screen shots below for more details on this.
You can do this by placing the `.screen` div inside the `.btn-container` as illustrated below, also remove the `width` styling for the `.screen` div
```HTML
<div class="calculator-wrapper">
  <div class="btn-container">
    <div class="screen"><p>0</p></div>
  </div>
</div>
```

```CSS
.screen {
    height: 4rem;
    background-color: #444;
    font-size: $myFontSize;
    font-weight: $myFontWeight;
    text-align: right;
    padding: 10px 20px;
    color: aliceblue;
    box-shadow: $shadow;
  }
```
## Microverse design
<img src="https://user-images.githubusercontent.com/16392046/204849900-ab520fc4-3648-47d6-9d36-f999622976b5.png" />

## Current implementation
<img src="https://user-images.githubusercontent.com/16392046/204850153-e6a1c420-bf56-4ccf-9496-eeebfed28507.png" width="350px" />